src/js/game/LevelMVC/LevelBlock.js

```diff
@@ -172,7 +172,7 @@ module.exports = class LevelBlock {
       this.isEntity = true;
     }
 
-    if (blockType.startsWith("bed")) {
+    if (blockType === "bedFoot" || blockType === "bedHead") {
```

Comment:
As as aside, this is exactly why I hate that we're relying so much on string comparisons for all this

Reply:
Agreed!

Reply:
Does this change anything? Are there any other blocks other than Foot and Head that begin with 'bed' string?

Reply:
Yes: "bedrock".